image_client/picturae_importer.py

```diff
@@ -32,14 +30,18 @@ def __init__(self, paths, date_string=None):
         self.date_use = date_string
 
         self.logger = logging.getLogger('PicturaeImporter')
+
         # full collector list is for populating existing and missing agents into collector table
         # new_collector_list is only for adding new agents to agent table.
         empty_lists = ['barcode_list', 'image_list', 'full_collector_list', 'new_collector_list',
-                       'taxon_list']
+                       'taxon_list', 'new_taxa']
```

Comment:
created new_taxa list to store new taxa for picbatch db, taxon_list gets overwritten every row so naturally didn't work